src/site/content/en/blog/bluetooth/index.md

```diff
@@ -0,0 +1,549 @@
+}
+```
+
+You can also call `device.gatt.disconnect()` to disconnect your web app from the
+Bluetooth device. This will trigger existing `gattserverdisconnected` event
+listeners. Note that it will NOT stop bluetooth device communication if another
+app is already communicating with the Bluetooth device. Check out the [Device
+Disconnect Sample] and the [Automatic Reconnect Sample] to dive deeper.
+
+{% Aside 'caution' %}
+Bluetooth GATT attributes, services, characteristics, etc. are invalidated when
+a device disconnects. This means your code should always retrieve (through
+`getPrimaryService(s)`, `getCharacteristic(s)`, etc.) these attributes after
+reconnecting.
+{% endAside %}
+
+### Read and write to Bluetooth descriptors
+
+Bluetooth GATT descriptors are attributes that describe a characteristic value.
+You can read and write them to in a similar way to Bluetooth GATT
+characteristics.
+
+Let's see for instance how to read the user description of the measurement
+interval of the device's health thermometer.
+
+In the example below, `health_thermometer` is the [Health Thermometer service],
+`measurement_interval` the [Measurement Interval characteristic], and
+`gatt.characteristic_user_description` the [Characteristic User Description
+descriptor].
+
+```js/4-9
+navigator.bluetooth.requestDevice({ filters: [{ services: ['health_thermometer'] }] })
+.then(device => device.gatt.connect())
+.then(server => server.getPrimaryService('health_thermometer'))
+.then(service => service.getCharacteristic('measurement_interval'))
+.then(characteristic => characteristic.getDescriptor('gatt.characteristic_user_description'))
+.then(descriptor => descriptor.readValue())
+.then(value => {
+  const decoder = new TextDecoder('utf-8');
+  console.log('User Description: ' + decoder.decode(value));
+})
+.catch(error => { console.log(error); });
+````
+
+Now that we've read the user description of the measurement interval of the
+device's health thermometer, let's see how to update it and write a custom
+value.
+
+```js/5-9
+navigator.bluetooth.requestDevice({ filters: [{ services: ['health_thermometer'] }] })
+.then(device => device.gatt.connect())
+.then(server => server.getPrimaryService('health_thermometer'))
+.then(service => service.getCharacteristic('measurement_interval'))
+.then(characteristic => characteristic.getDescriptor('gatt.characteristic_user_description'))
+.then(descriptor => {
+  const encoder = new TextEncoder('utf-8');
+  const userDescription = encoder.encode('Defines the time between measurements.');
+  return descriptor.writeValue(userDescription);
+})
+.catch(error => { console.log(error); });
+```
+
+## Samples, Demos and Codelabs
+
+All [Web Bluetooth samples] below have been successfully tested. To enjoy these
+samples to their fullest, I recommend you install the [BLE Peripheral Simulator
+Android App] which simulates a BLE peripheral with a Battery Service, a Heart Rate
+Service, or a Health Thermometer Service.
```

Comment:
As @jpmedley mentioned in the USB article ([comment](https://github.com/GoogleChrome/web.dev/pull/4095/files#r508851245)), are we still confident that all the samples/codelabs/etc. work?

Reply:
Yep. I maintain Web Bluetooth samples, demos, and codelabs